### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by henrydays**
enable the tool to analyze history in a selected range of commits. Useful in react due to it's size and difficulty to analyze each commit

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -121,27 +135,82 @@ function getHistory() {
...
-                            if (lastCheckSum !== getPackageChecksum(local)) {
-                                // works synchronously in order to ensure that the
-                                // pc would not crash due to too many npm installs at the same time
-                                installRepoSync(repo, local);
-                                lastCheckSum = getPackageChecksum(local);
-                            }
-                            getMetricsForRepo(repo, local).then((res) => {
-                                if (_.isObject(res)) {
-                                    res.hash = commitHash;
+                if (_.isNumber(argv.de) && _.isNumber(argv.a)) {
```
Comment (lfac-pt) [inline src/index.js]: What is `de` and `a`?

File: src/index.js [hunk 2]
```diff
@@ -121,27 +135,82 @@ function getHistory() {
...
+                        }
                     }
                 }
+
+                else if (_.isString(argv.commits)) {
+                    for (var i = 0; i < commitHashes.length; i++) {
+                        if ((i < commitHashes.length) && containsCommit(i)) {
+                            const commitHash = commitHashes[i];
+                            funcs.push(function (callback) {
+                                console.log("fez push")
```
Comment (lfac-pt) [inline src/index.js]: Remove this?

Comment (henrydays) [reply src/index.js]: already solved in previous commit

File: src/index.js (hunk 1 above)
Comment (henrydays) [reply src/index.js]: same

File: src/index.js [hunk 3]
```diff
@@ -121,27 +135,81 @@ function getHistory() {
...
-                            if (lastCheckSum !== getPackageChecksum(local)) {
-                                // works synchronously in order to ensure that the
-                                // pc would not crash due to too many npm installs at the same time
-                                installRepoSync(repo, local);
-                                lastCheckSum = getPackageChecksum(local);
-                            }
-                            getMetricsForRepo(repo, local).then((res) => {
-                                if (_.isObject(res)) {
-                                    res.hash = commitHash;
+                if (_.isNumber(argv.from) && _.isNumber(argv.to)) {
```
Comment (lfac-pt) [inline src/index.js:138]: Can you just add some comments explaining each of the ifs?

**Pull request merged**